src/peerbook/libp2p_peerbook.erl

```diff
@@ -88,8 +88,7 @@ put(#peerbook{tid=TID, stale_time=StaleTime}=Handle, PeerList0, Prevalidated) ->
             false -> lists:filter(fun libp2p_peer:verify/1, PeerList0)
         end,
     ThisPeerId = libp2p_swarm:pubkey_bin(TID),
-    %% XXX uncomment this to reject any peers publishing RFC1918 addresses once the network has transitioned over
-    AllowRFC1918 = true, %% is_rfc1918_allowed(TID),
+    AllowRFC1918 = is_rfc1918_allowed(TID),
```

Comment:
We should check we don't have many peers advertising RFC1918 addresses before we merge this.

Reply:
I'll check that tomorrow morning.